lib/manageiq/release/backport_prs.rb

```diff
@@ -39,16 +39,25 @@ def backport_prs
           puts
           puts "** #{pr.html_url}".cyan.bold
 
-          success = backportable?(pr) && backport_pr(pr.number, pr.user.login)
-          puts
+          pr_number = pr.number
+          pr_author = pr.user.login
 
-          if success
-            repo.git.log("-1")
-            puts
+          if already_on_branch?(pr_number)
+            handle_already_on_branch(pr_number)
+
+            puts "The commit already exists on the branch. Skipping.".yellow
           else
-            puts "A conflict was encountered during backport.".red
-            puts "Stopping backports for #{github_repo}.".red
-            break
+            success = backportable?(pr) && backport_pr(pr_number, pr_author)
```

Comment:
I find it odd that `backport_pr` is passed in a number and userid, instead of a PR (just like backportable? now).  But that can be refactored in a future PR.